src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -1198,6 +1223,14 @@ public async Task LoadExtendedItemPropertiesAsync(ListedItem item)
 								SetFileTag(item);
 							});
 						}
+						else
+						{
+							// Load cached thumbnail for cloud files
+							if (item.SyncStatusUI.SyncStatus != CloudDriveSyncStatus.NotSynced && item.SyncStatusUI.SyncStatus != CloudDriveSyncStatus.Unknown)
+								_ = LoadCachedThumbnailAsync(item);
+							else
+								_ = LoadThumbnailAsync(item);
+						}
```

Comment:
Should this not be executed if `wasSyncStatusLoaded` is false? More to the point, I think this doesn't need to be executed after the properties are loaded, but could be executed immediately after the basic icon has been loaded.

Reply:
> but could be executed immediately after the basic icon has been loaded.

We can execute `LoadThumbnailAsync` at that point, but then we'd also be executing `LoadCachedThumbnailAsync` for cloud files once sync status is loaded.

Reply:
I see.

Reply:
If you think that's fine, then we can go ahead and do make this change.

Reply:
I made additional changes that I think resolve this thread